### ISSUE TITLE
moge v2，fine turning problem

### ISSUE THREAD

**Issue by 3sbase** [microsoft/MoGe#135]
moge v2，fine turning。We used our own dataset to fine-tune the model, which, together with the pre-trained model, performed depth estimation on the same image, yielding identical results. What went wrong?

**Comment by Crigges** [comment]
Hey, 

we also try to finetune the moge2 model using the map-anything framework. But during the training, the loss basically stays constant. Have you made any progress so far?
